im.nheko.Nheko.yaml

```
@@ -0,0 +1,247 @@
+id: im.nheko.Nheko
+command: im.nheko.Nheko
+runtime: org.kde.Platform
+runtime-version: '6.6'
+sdk: org.kde.Sdk
+finish-args:
+  - --device=dri
+  # needed for webcams, see #517
+  - --device=all
+  - --share=ipc
+  - --share=network
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --socket=x11
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.freedesktop.secrets
+  - --talk-name=org.freedesktop.StatusNotifierItem
+  # needed for tray icon
+  - --talk-name=org.kde.*
```

```suggestion
  - --talk-name=org.kde.StatusNotifierWatcher
```

https://github.com/flathub/io.github.NhekoReborn.Nheko/blob/59d2532add48f8b85536253ea8c0c52e080e0c33/io.github.NhekoReborn.Nheko.yaml#L17C17-L17C46

This should not be needed with recent runtimes, you just need to make sure it is not using `StatusNotifierItem-$PID-$ID`

---

If `org.kde.*` is needed, this needs an exception https://docs.flathub.org/docs/for-app-authors/linter#finish-args-wildcard-kde-own-name

But QT should not use it anymore. This is supposed to be fixed with Qt 6

---

If you remove this, you will likely need to add the kwalletd interfaces (kde6 and 5 individually)

---

Well, let's see if these permissions work. I wasn't aware that Qt6 removed the need for this and couldn't find documentation for it anywhere either.

---

Tray seems to be working for me.